index.html

```diff
@@ -1,13 +1,208 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+  <!-- font style  -->
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link
+    href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,900;1,100;1,300;1,400;1,700&display=swap"
+    rel="stylesheet">
+
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.5.0/font/bootstrap-icons.css">
+
+
   <link rel="stylesheet" href="style.css">
   <title>WeLead conferences</title>
 </head>
+
 <body>
+  <main>
+    <header>
+
+      <div class="header-icon">
+        <ul>
+          <li><a href="./index.html"><i class="bi bi-facebook"></i></a></li>
+          <li><a href="./index.html"><i class="bi bi-twitter"></i></a></li>
+          <li><a href="./index.html">English</a></li>
+          <li><a href="./index.html">My Page</a></li>
+          <li><a href="./index.html">Logout</a></li>
+        </ul>
+      </div>
+
+      <nav class="nav-container ">
+
+        <a href="./index.html" class="logo"><span>WL</span> &nbsp; WL Summit <br>2023</a>
+
+
+        <ul class="nav-menu">
+
+          <li class="nav-item">
+
+            <a href="#home" class="nav-link" id="home-link">Home</a>
+
+          </li>
+
+          <li class="nav-item">
+
+            <a href="./about.html" class="nav-link" id="about-link">About</a>
+
+          </li>
+
+          <li class="nav-item">
+
+            <a href="#program" class="nav-link" id="program-link">Program</a>
+
+          </li>
+          <li class="nav-item">
+
+            <a href="#join" class="nav-link" id="join-link">Join</a>
+
+          </li>
+          <li class="nav-item">
+
+            <a href="#speaker" class="nav-link" id="speaker-link">Speaker</a>
+
+          </li>
+          <li class="nav-item">
+
+            <a href="#campaign" class="nav-link" id="campaign-link">Campaign</a>
+
+          </li>
+
+        </ul>
+
+        <div class="hamburger">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+        </div>
+
+      </nav>
+    </header>
+
+    <section class="">
+      <div class="first-section">
+        <section class="title container">
+          <p class="title-paragraph">"WeLead Political Summit"</P>
+          <h1>TO REFORM THE NEED FOR INCREASING <br> WOMEN’S ROLE IN POLITICS <br>2023</h1>
```

Comment:
Kindly take note of this for both of your pages😉😉😉 Your webpage should have just one `h1` tag which serves as the intro header😉😉 you can now make use of the `h2`, `h3`  etc subsequently😉😉 take note of the hierarchy when making use of the `h` tag. An `h1` should be the main heading, then the `h2` should be used subsequently. If you have a heading under an `h2` tag, make use of an `h3`. if you have a heading under an `h3` tag, make use of an h4 and so on🤗🤗 This is semantically appropriate and it aids accessibility for screen readers😉😉 kindly fix this everywhere it is needed🤗🤗

Reply:
thank you very much for the reminder am on it right away

Reply:
kk

Reply:
done